harvester/harvester/settings.py

```diff
@@ -355,6 +355,75 @@ def strip_sensitive_data(event, hint):
     'application/x-java': 'app',
 }
 
+MIME_TYPE_TO_TECHNICAL_TYPE = {
+    'unknown': 'unknown',
+    'application/pdf': 'document',
```

Comment:
is het niet makkelijker om een mime type te laten wijzen naar een dict? Dus iets van:

```
{
  'application/pdf': {
    'file_type': 'pdf',
    'technical_type: 'document'
  },
  'application/x-pdf': {
  ... etc
}
```

Zodat je bij een eventueel nieuw mime-type niet het op twee plekken hoeft toe te voegen

Reply:
Het idee is dat we snel de file_type gaan weggooien hierna. Dus ik heb het even bewust snel gecopypasted en als productie over is op het nieuwe type dan kan ik file_type verwijderen.

Reply:
Ah ok, laat dan maar inderdaad 😉